.gitignore

```diff
@@ -5,3 +5,4 @@ tivan
 .idea
 *~
 *#
+.vscode
```

Comment:
Can you remove this, you could put it into a `~/.gitignore` file and add an excludesfile to your `~/.gitconfig`.  I'm going to remove some of the other entries in here that don't belong.